CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,115 @@
+# Contribution Guide
+
+---
+
+Thank you for being interested in contributing to Aragon! 🎉  We’re excited to have you building this vision with us.
+
+There are many ways to contribute! Some ideas are:
+
+- writing tutorials, blog posts or improving the documentation,
+- submitting bug reports,
+- submitting feature requests,
+- refactoring code,
+- or even writing new functionality to incorporate into the project.
+
+All members of our community are expected to follow our [Code of Conduct](https://discord.com/channels/672466989217873929/953988031713534022/1037740253080997918). Please make sure you are welcoming and friendly in all of our spaces. This is a non-negotiable 💙
+
+## Your first contribution
+
+Unsure where to begin contributing to Aragon?
+
+You can start with a [Good First Issue](https://github.com/aragon/developer-portal/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
```

Comment:
Good in theory, but in practice, we do it differently.
We are not using GH issues but JIRA for task planning. Accordingly external contributors won't see the tasks we are planning to do. The issue list is mostly empty for all our repos. There won't be labels with "Good first issue".

What's the plan for having good first issues here @Rekard0?